scripts/Logistics-Pipes.zs

```diff
@@ -0,0 +1,873 @@
+
+// --- Logistics System Entrance Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSystemEntranceLogistics>);
+
+// --- Logistics System Entrance Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSystemDestinationLogistics>);
+
+// --- Logistics Inventory System Connector
+recipes.remove(<LogisticsPipes:item.PipeItemsInvSysConnector>);
+
+
+// ||||| Fluid Pipes |||||
+
+
+// --- Logistics Basic Fluid Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidBasic>);
+
+// --- Logistics Fluid Container Supplier
+recipes.remove(<LogisticsPipes:item.PipeItemsFluidSupplier>);
+
+// --- Logistics Fluid Supplier
+recipes.remove(<LogisticsPipes:item.PipeFluidSupplierMk2>);
+
+// --- Logistics Fluid Provider Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidProvider>);
+
+// --- Logistics Fluid Request Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidRequestLogistics>);
+
+// --- Logistics Fluid Satellite Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidSatellite>);
+
+// --- Logistics Fluid Insertion Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidInsertion>);
+
+// --- Logistics Fluid Extractor Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidExtractor>);
+
+
+// ||||| Chassis |||||
+
+
+// --- Logistics Chassi MK1
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk1>);
+
+// --- Logistics Chassi MK2
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk2>);
+
+// --- Logistics Chassi MK3
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk3>);
+
+// --- Logistics Chassi MK4
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk4>);
+
+// --- Logistics Chassi MK5
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk5>);
+
+
+
+// *======= Adding Back Recipes =======*
+// --- Assembler Recipes --- 
+
+// - LV Tier
+// ||||| Blocks |||||
+// --- Power Junction
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 1152, 800, 30);
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.tin> * 576, 800, 30);
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 288, 800, 30);
+
+// --- Unrouted Logistics Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsBasicTransport> * 2, [<ore:stickLongSteel> * 2, <bartworks:BWPumpParts>, <ore:wireFineGold> * 2, <gregtech:gt.integrated_circuit:18> * 0], null, 100, 30);

```

Comment:
This is unrouted logistics pipe that can't do anything on its own. Metal + fine gold wire is enough expensive.

Reply:
I wanted some kind of glass pipe or pane for the pipes, Observe that you get more efficient recipes when you have advanced a little bit into LV.